### DISCUSSION TITLE
Whiteboard visualization

### DISCUSSION THREAD

**Discussion by intelagense** [Together-100Devs/Together#9] [General]
![image](https://user-images.githubusercontent.com/18331781/200149045-6d691efd-06a7-4a73-abc1-d0f26a86a257.png)

**Comment by Caleb-Cohen** [comment]
@intelagense Let's add a contact to the event view as well. It can just be a discord name, but users should have someone they contact that's responsible for the non-official events like standup and normal class. I've started on adding schema at #12 and have a branch called Schema attempting this model.

**Comment by intelagense** [comment]
![image](https://user-images.githubusercontent.com/18331781/200369304-a52bdae6-5ef9-4bbe-ba2b-d06651d19f70.png)

It is not super polished but here is an updated whiteboard with a user profile view added as well.

**Reply by Caleb-Cohen** [reply]
Looks good to me. @Natashka333 if you have the time to create a design mockup of the event view, add event view, and profile that would be super awesome.

**Reply by yuhwaa** [reply]
I can work on updating the userSchema based on what we have so far.

**Reply by Caleb-Cohen** [reply]
> I can work on updating the userSchema based on what we have so far.

That's a good catch. I'll add that to the readme. Can you open a issue for discussion?
